sdks/go/opspec/ref_test.go

```diff
@@ -0,0 +1,19 @@
+package opspec
+
+import (
+	"testing"
+
+	. "github.com/onsi/gomega"
+)
+
+func TestRefToName(t *testing.T) {
```

Comment:
Just a nit but for consistency might be nice to have wire up same as all the other tests; I also retroactively noticed this variance in https://github.com/opctl/opctl/blob/c5611ff2884eef74c2531064ec8776bea0eaf739/sdks/go/internal/aggregate_error/aggregate_error_test.go#L12 which I was actually hoping to cleanup. It's one of those things I'd hope we just do one way; the way we choose doesn't seem super important (unless it is to you); moreso just find value in there being a single way used throughout.

Reply:
It's a far nicer IDE experience to use standard go test patterns - I can debug a single test without requiring running the entire suite.

Reply:
![ref_test go — opctl - Code - 2021-03-22 at 10 37 27](https://user-images.githubusercontent.com/329222/111969506-9c14d480-8afa-11eb-83fc-601fafd1d82e.png)

Reply:
I had to add the [VSCode launch task](https://github.com/opctl/opctl/blob/6b4e1974c70f80c561110ab873d506ef734b7c62/.vscode/launch.json#L27) to do test debugging because of Ginkgo, and it's a lot more overhead since it runs all tests in the suite.